AmericanMedicalAssociation/joe-style-guide
# Move Assets From Theme To Styleguide

## JIRA Ticket(s)
N/A - Developer Task

## Description:
- added SG assets relocated from theme directory (PR: https://github.com/AmericanMedicalAssociation/code-of-ethics/pull/255) to remain consistent with other AMA site structures

## To Test:
- Follow testing steps in PR: https://github.com/AmericanMedicalAssociation/code-of-ethics/pull/255
- Verify site displays without style issues when linking to local SG

## Automated Test:
N/A

## Style Guide:
N/A

## Relevant Screenshots/GIFs:
N/A

## Additional Notes:
N/A
---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
